### PR TITLE
Add GetDcLocations Request

### PR DESCRIPTION
**Pull request by Yisaer**

### CODE REVIEW

File: proto/pdpb.proto [hunk 1]
```diff
@@ -77,6 +77,8 @@ service PD {
     rpc GetOperator(GetOperatorRequest) returns (GetOperatorResponse) {}
 
     rpc SyncMaxTS(SyncMaxTSRequest) returns (SyncMaxTSResponse) {}
+
+    rpc GetDcLocations(GetDcLocationsRequest) returns (GetDcLocationsResponse) {}
```
Comment (JmPotato) [inline proto/pdpb.proto:81]: I think `GetDCLocations` is better.

File: proto/pdpb.proto [hunk 2]
```diff
@@ -630,3 +632,12 @@ message SyncMaxTSResponse {
...
     repeated string dcs = 3;
 }
+
+message GetDcLocationsRequest {
+    RequestHeader header = 1;
+}
+
+message GetDcLocationsResponse {
+    ResponseHeader header = 1;
+    repeated string dcLocations = 2;
```
Comment (JmPotato) [inline proto/pdpb.proto:642]: The snake case should be used here: `dc_locations`.

**Review by JmPotato — APPROVED** [review]
LGTM

**Review by HunDunDM — APPROVED** [review]
LGTM

**Pull request merged**